services/ux-backend/main.go

```diff
@@ -91,6 +102,10 @@ func main() {
 	http.HandleFunc("/rotate-keys", func(w http.ResponseWriter, r *http.Request) {
 		rotatekeys.HandleMessage(w, r, cl)
 	})
+	http.HandleFunc("/remove-ceph-client", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
we are cleaning up consumer and to do so we need to remove some resources which are internal and may not be cephclient only, can the ep be `/cleanup-consumer`?

Reply:
Yes instead of using remove, cleanup sounds better

Reply:
I think there is a misconception here. With rest-based APIs the path indicates a resource or a ser of resources.
There is no resource in the system that is named `cleanup-consumer`